package.json

```diff
@@ -90,6 +89,8 @@
   },
   "devDependencies": {
     "@commitlint/cli": "^6.0.1",
+    "babel-plugin-flow-react-proptypes": "^10.0.0",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
```

Comment:
These need to be under `dependencies`, not `devDependencies`, because consumers are meant to install them too.